data/sql/updates/pending_db_world/rev_1556033308686302400.sql

```diff
@@ -0,0 +1,232 @@
+(17523, 0, 0, 0, 0, 0, 100, 0, 1500, 1500, 33000, 33000, 11, 31288, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Myst Leecher - In Combat - Cast \'31288\'');
+
+
+-- Creating a new SmartAI script for [Creature] ENTRY 17608 (name: Sunhawk Pyromancer)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17608;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17608);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17608, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 31734, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Pyromancer - On Aggro - Cast \'31734\''),
+(17608, 0, 1, 0, 0, 0, 100, 0, 1500, 1500, 25700, 25700, 11, 11962, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Pyromancer - In Combat - Cast \'11962\''),
+(17608, 0, 2, 0, 0, 0, 100, 0, 5200, 5200, 9900, 9900, 11, 9053, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Pyromancer - In Combat - Cast \'9053\'');
+
+
+-- Creating a new SmartAI script for [Creature] ENTRY 17607 (name: Sunhawk Defender)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17607;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17607);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17607, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 31734, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Defender - On Aggro - Cast \'31734\''),
+(17607, 0, 1, 0, 0, 0, 100, 0, 3200, 3200, 15400, 15400, 11, 15284, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Defender - In Combat - Cast \'15284\''),
+(17607, 0, 2, 0, 0, 0, 100, 0, 8600, 8600, 34200, 34200, 11, 31737, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Defender - In Combat - Cast \'31737\'');
```

Comment:
Here too, if you can add one more ability [Backhand](https://wotlk.evowow.com/?spell=6253)